app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -24,81 +26,87 @@ class ProjectBoard extends React.Component {
     this.props.fetchProjectBoard(this.props.projectId);
   }
 
+  onAddStory = (status) => {
```

Comment:
If you wanna you can make this function returns a new function, just o keep `renderColumnAction` simpler:
```js
onAddStory = (status) => () => {
  ...
}

renderColumnAction = (status) =>
    <AddStoryButton onAdd={this.onAddStory(status)}/>
```